### PR TITLE
Solaris 10u11 - zfs base box

### PR DESCRIPTION
**Pull request by scotthain**
Both Fusion and Virtualbox supported

### CODE REVIEW

File: packer/floppy/solaris10/S99startup.sh [hunk 1]
```diff
@@ -19,6 +19,8 @@ perl -pi -e 's/vagrant:UP/vagrant:VrvarmJYR3SHs/g' /etc/shadow
 # defaults to a locked account, unlock it
 passwd -u vagrant
 
+echo "vagrant user created Installed" >> /tmp/s99log
```
Comment (juliandunn) [inline packer/floppy/solaris10/S99startup.sh]: Nitpick, either choose "created" or "Installed" not both?

File: packer/floppy/solaris10/S99startup.sh [hunk 2]
```diff
@@ -29,27 +31,26 @@ chmod -R go-rwsx /home/vagrant/.ssh
 echo GSSAPIAuthentication no >> /etc/ssh/sshd_config
 echo LookupClientHostnames no >> /etc/ssh/sshd_config
 
-# install pkgutil because yay
-/usr/sfw/bin/wget --no-check-certificate 'http://get.opencsw.org/now' -O /tmp/pkgutil.pkg
+echo "SSH Installed" >> /tmp/s99log
+
+# install omnibus-build-essential package
+cd /tmp
 echo "mail=\ninstance=overwrite\npartial=nocheck\nrunlevel=nocheck\nidepend=nocheck\nrdepend=nocheck\nspace=nocheck\nsetuid=nocheck\nconflict=nocheck\naction=nocheck\nbasedir=default" > /tmp/noask
```
Comment (juliandunn) [inline packer/floppy/solaris10/S99startup.sh]: I didn't ask last time, but is it worth doing a `rm -f /tmp/noask` at the end of this script?

**Comment by juliandunn** [comment]
Looking good... have we decided just to get rid of the 10u5 definition? We don't need it for anything now?

**Comment by scotthain** [comment]
@juliandunn the u5 def is no longer needed, as we have mapfile support to build on anything 10u1+. I have a few more tweaks before this is 100% ready (Including those you mentioned. Damn you copy paste!).

File: packer/floppy/solaris10/S99startup.sh (hunk 2 above)
Comment (scotthain) [inline packer/floppy/solaris10/S99startup.sh]: The final reboot cleans /tmp. I think rm -f is overkill in this case.

**Comment by scotthain** [comment]
@juliandunn pending CI, should be good to merge if you're cool with it.

**Comment by juliandunn** [comment]
:+1: :shipit:

File: packer/floppy/solaris10/S99startup.sh (hunk 2 above)
Comment (curiositycasualty) [inline packer/floppy/solaris10/S99startup.sh]: `echo -e` for "\n" translation?

Comment (scotthain) [inline packer/floppy/solaris10/S99startup.sh]: Unneeded with Solaris 10 echo.

**Pull request merged**